test/geocoders/nominatim.py

```diff
@@ -207,3 +207,20 @@ def test_missing_reverse_details(self):
             'address',
             res
         )
+
+    def test_view_box(self):
+        box_tuple = tuple([180, 22, -180, -22])
```

Comment:
Another suggestion would be to turn this to a for loop, instead of copy-pasting the same code but with different formats of the view_box. 

Also it's more preferable to patch `self.geocoder` and use `self.geocode_run` instead of asserting the result manually. As `geocoder` is a class attribute, and modifying `self.geocoder` would change it on an instance rather than the class, you can simply assign it in the test and add a tearDown method which would delete the instance value to use the class's one again:
```
def tearDown(self):
    try:
        del self.geocoder
    except AttributeError:
        pass
```

A complete example:
```
    self.geocode_run(...)  # Show that without view_box the result is different.
    for view_box in [(180, 22, -180, -22),
                     [Point(22, 180), Point(-22, -180)],
                     ...]:
        self.geocoder = Nominatim(view_box=view_box)
        self.geocode_run(...)
```

I'm not sure if this is the best way, but for now it looks reasonable to me. What do you think?